canonical-websites/www.canonical.com
# Remove unused career templates

## Done

Remove unused career templates

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8002/](http://0.0.0.0:8002/)
4. Make sure career templates that aren't used have been removed 

## Issue / Card
Fixes #263
